app/models/content_configuration.rb

```diff
@@ -69,4 +69,7 @@ class ContentConfiguration < Spree::Preferences::FileConfiguration
 EOS
 
   preference :footer_about_url, :string, default: "http://www.openfoodnetwork.org/ofn-local/open-food-network-australia/"
+
+  #User Guide
+  preference :user_guide_link, :string, default: 'http://www.openfoodnetwork.org/platform/user-guide/'
```

Comment:
I just followed that link and the current URL is https://guide.openfoodnetwork.org/. We should use that here.